boards/arm64/qemu_cortex_a53/qemu_arm64_kvm_defconfig

```diff
@@ -0,0 +1,20 @@
+CONFIG_SOC_QEMU_CORTEX_A53=y
```

Comment:
This is not making much sense to me. KVM virt is not Cortex-A53. If any, you should add a separate "virt" SoC type.